scripts/docker/buildpack-deps/Dockerfile.ubuntu1604.clang.ossfuzz

```diff
@@ -113,6 +113,22 @@ RUN set -ex; \
 	ninja install/strip; \
 	rm -rf /usr/src/hera
 
+# gmp
+RUN set -ex; \
+    # Replace system installed libgmp static library
+    # with sanitized version
+    rm -f /usr/lib/x86_64-linux-gnu/libgmp.a; \
```

Comment:
Maybe remove via `apt` if it's system installed?

Reply:
Done!

Reply:
Turns out `apt -y remove libgmp-dev` also removes `mlton` (a dependency) that we need to build libabicoder later on. Reverting to the manual replacement for now.

Reply:
I'm just mainly concerned that the lib is installed somewhere else and this wouldn't remove it, creating problems later.
Do you think the manual removal is better than
```
apt -y remove libgmp-dev
apt -y install mlton
```
?

Reply:
Yeah, I thought of this as well but the problem is the mlton installation pulls in libgmp as well!

Reply:
Cyclic dependencies, sigh!

Reply:
Ohh that's annoying. Well, if you're confident that this will be the installed path, I'd say it's fine